vizro-core/changelog.d/20240305_142454_huong_li_nguyen_fix_card_validation.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Changed
+
+- A bullet item for the Changed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Deprecated
+
+- A bullet item for the Deprecated category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Fixed
+
+- Consolidate validation for `Card.href` and `Page.path`. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

Comment:
```suggestion
- Allow spaces in `Card.href` and `Page.path` strings - e.g. `Page Title` now works as reference to a page. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

Reply:
Spaces did work already in `Page.path`, so I think I wouldn't take over the first part. The second part I've split up in a separate changelog entry now, as this is changed functionality and doesn't belong to fixed. I agree that it's worth mentioning 👍

Reply:
Ok cool! I would actually remove the fixed bit then as I think that it is not really interesting to a user who needs to decide if it's worth upgrading Vizro to a newer version. Less is more here I feel :)

Reply:
For me, these are actually two different points. The fixed one does fix a bug that has been reported by Alexey and was the main point for this PR: https://github.com/mckinsey/vizro/issues/285

The fact that you can provide a `Page.title` directly to `Card.href` is an add-on as I got rid of the requirement having to provide a `/` + the point above.